app/views/notes/new.html.erb

```diff
@@ -5,9 +5,17 @@
 <div class="note">
   <p class="alert alert-info"><%= t(".intro") %></p>
   <% if !current_user %>
-    <p class="alert alert-warning"><%= t ".anonymous_warning_html",
-                                         :log_in => link_to(t(".anonymous_warning_log_in"), login_path(:referer => new_note_path)),
-                                         :sign_up => link_to(t(".anonymous_warning_sign_up"), new_user_path) %></p>
+    <p class="alert alert-warning">
+      <%= t ".anonymous_warning_html",
+            :log_in => link_to(t(".anonymous_warning_log_in"), login_path(:referer => new_note_path)),
+            :sign_up => link_to(t(".anonymous_warning_sign_up"), new_user_path) %>
+    <% if @anonymous_notes_count >= 10 %>
```

Comment:
You have a `>=` condition here, but the message reads `You have already posted more than %{x_anonymous_notes}`. If I create exactly 10 notes, it will tell me that I "posted more than 10".